lib/getNpmPluginConfig.js

```diff
@@ -0,0 +1,30 @@
+const { isArray, isPlainObject, isNil } = require("lodash");
+
+/**
+ * Get the options of @semantic-release/npm plugin.
+ *
+ * @param {MultiContext} context The multi-semantic-release context.
+ * @returns {object} The @semantic-release/npm plugin options if any, undefined otherwise.
+ *
+ * @internal
+ */
+function getNpmPluginConfig(context) {
+	if (context.options.plugins) {
+		for (let i = 0; i < context.options.plugins.length; i++) {
+			const plugin = context.options.plugins[i];
+			let path = "";
+			let config = {};
+			if (isArray(plugin)) {
+				[path, config] = plugin;
+			} else if (isPlainObject(plugin) && !isNil(plugin.path)) {
+				({ path, ...config } = plugin);
+			}
+			if (["@semantic-release/npm", "@semrel-extra/npm"].includes(path)) {
```

Comment:
Let's look for 'pkgRoot' option for any plugin instead? And then compare the value with '.'